<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DirFindFile</class>
 <widget class="QWidget" name="DirFindFile">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>389</width>
    <height>165</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="maximumSize">
   <size>
    <width>400</width>
    <height>200</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>DirFindFile</string>
  </property>
  <property name="windowIcon">
   <iconset resource="RealCompare.qrc">
    <normaloff>:/Resources/img/main.png</normaloff>:/Resources/img/main.png</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Find Options</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Find File Name</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="fileNameLineEdit"/>
        </item>
       </layout>
      </item>
      <item row="1" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QRadioButton" name="findLeftRadioButton">
          <property name="text">
           <string>find in left</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="findRightRadioButton">
          <property name="text">
           <string>find in right</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="caseSensitiveCheckBox">
          <property name="text">
           <string>case sensitive</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QPushButton" name="pushPrevButton">
       <property name="text">
        <string>Find Prev</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushNextButton">
       <property name="text">
        <string>Find Next</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushCloseButton">
       <property name="text">
        <string>Close</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="RealCompare.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>pushPrevButton</sender>
   <signal>clicked()</signal>
   <receiver>DirFindFile</receiver>
   <slot>slot_findPrev()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>88</x>
     <y>129</y>
    </hint>
    <hint type="destinationlabel">
     <x>126</x>
     <y>156</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushNextButton</sender>
   <signal>clicked()</signal>
   <receiver>DirFindFile</receiver>
   <slot>slot_findNext()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>227</x>
     <y>124</y>
    </hint>
    <hint type="destinationlabel">
     <x>259</x>
     <y>161</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushCloseButton</sender>
   <signal>clicked()</signal>
   <receiver>DirFindFile</receiver>
   <slot>hide()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>341</x>
     <y>123</y>
    </hint>
    <hint type="destinationlabel">
     <x>387</x>
     <y>107</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>slot_findPrev()</slot>
  <slot>slot_findNext()</slot>
 </slots>
</ui>
